pyproject.toml

```
@@ -0,0 +1,60 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools>=61.2",
+]
+
+[project]
+name = "python-docs-bootstrapper"
+version = "0.1.1"
+description = "Bootstrapper for Python documentation translations"
+readme = "README.md"
+keywords = [
+  "automation",
+  "CLI",
+  "documentation",
+  "i18n",
+  "python-docs",
+  "sphinx",
+  "translation",
+  "utilities",
+]
+license = {text = "MIT"}
+authors = [{name = "egeakman", email = "me@egeakman.dev"}]
+requires-python = ">=3.10"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Console",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: POSIX :: Linux",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Utilities",
+]
+dependencies = [
+  "sphinx==4.5",
+]
+[project.urls]
+Download = "https://github.com/egeakman/python-docs-bootstrapper/archive/0.1.1.tar.gz"
```

Is this supported?

```suggestion
Download = "https://github.com/egeakman/python-docs-bootstrapper/archive/{project.version}.tar.gz"
```

---

Tried it, it is not. Is there a workaround or an alternative for variable interpolation?

---

No, it leaves it as `{project.version}.tar.gz"`

Actually, I never use "Download" here, the usual thing is for people to install via PyPI.

Perhaps something like this instead?

```suggestion
Releases = "https://github.com/egeakman/python-docs-bootstrapper/releases"
```

---

Makes sense, thanks!